test/mockupdb/test_network_disconnect_primary.py

```diff
@@ -26,7 +26,7 @@ def test_network_disconnect_primary(self):
         # Application operation fails against primary. Test that topology
         # type changes from ReplicaSetWithPrimary to ReplicaSetNoPrimary.
         # http://bit.ly/1B5ttuL
-        primary, secondary = servers = (MockupDB() for _ in range(2))
```

Comment:
Thanks for fixing this! Do you mind refactoring the code to remove "servers" altogether? This change was made by an automated tool and I'm hoping to avoid hitting the same issue the next time we run pyupgrade/ruff.

Reply:
or alternatively you could do:
```python
primary, secondary = servers = (MockupDB(), MockupDB())
```

Reply:
Thanks, I've removed `servers`, and the code is now consistent to the style of [test_handshake.py](https://github.com/mongodb/mongo-python-driver/blob/5831934b379e2a3c634778e92feed41837d4c9b2/test/mockupdb/test_handshake.py#L83-L88) and other similar tests. This issue will hopefully be avoided now.